docs/semgrep-cloud-platform/getting-started.md

```diff
@@ -94,6 +76,82 @@ Using Semgrep Supply Chain with Semgrep Cloud Platform provides users with the V
 
 </Tabs>
 
+If you are an **administrator** setting up Semgrep for your team, see [Adding an organization](#adding-an-organization).
+
+If you are a **team member joining an existing organization**, see [Joining an organization](#joining-an-organization).
+
+If you are a **personal user**, or simply want to run scans, see [Running scans](#running-scans).
+
+## Adding an organization
+
+This process creates an **organization account**, which your team members in GitHub or GitLab can join. Organization accounts share custom rules that are written by fellow team members, as well as records of findings and scans. 
+
+1. Sign in to Semgrep Cloud Platform. A dialog box appears.
+2. Click **Create an organization**.
+![DESCRIPTION](/img/create-an-org.png#md-width)
+3. Enter your **Organization display name**. This name typically corresponds to the name of your GitHub or GitLab org.
+4. Click **Create**.
+5. Your organization is created and you are prompted to start a scan. Instead of starting a scan, click **<i class="fa-solid fa-gear"></i> Settings** in the navigation menu.
+6. Click **Source code managers**.
+7. Select the source code manager that contains the org to connect to Semgrep Cloud Platform.
+![DESCRIPTION](/img/source-code-manager.png#md-width)
+8. Follow the steps to connect your source code manager. You can review permissions needed by Semgrep in [Requested permissions for GitHub and GitLab](#requested-permissions-for-github-and-gitlab).
+
+## Joining an organization
+
+:::info Prerequisites
+- Your admin or inviter should have an existing organization (org) in Semgrep that is connected to the org in either GitHub or GitLab.
+- You must be a member of the GitHub or GitLab org that is connected to Semgrep.
+:::
+
+For **members to join an organization**, perform the following steps to be added to the org:

```

Comment:
```suggestion
For **members**, perform the following steps to join the org:
```
I'm open to other alternative phrasing here, but the current "to join an organization...to be added to the org" feels heavy and redundant.